Gemfile.lock

```diff
@@ -74,10 +74,10 @@ GIT
 
 GIT
   remote: https://github.com/department-of-veterans-affairs/vets-json-schema
-  revision: c0f2a9bcf5f6eefd2182732732b1d6e17473db3d
+  revision: 897ad81be907fa6bd3d997f20ff1e0a912d436cd
   branch: master
   specs:
-    vets_json_schema (24.12.0)
+    vets_json_schema (24.12.1)
```

Comment:
Matches what's in master: https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/package.json#L3

Reply:
Looks like multiple PRs upgrading to 24.12.1 went into vets-json-schema at once

Reply:
yeah... I saw that too. @kjsuarez